YAFCui/ImGui/ImGuiLayout.cs

```diff
@@ -240,7 +241,10 @@ public void Dispose()
                 rect.Width += (padding.left + padding.right);
                 rect.Height += (padding.top + padding.bottom);
                 if (hasContent)
-                    gui.lastRect = gui.state.EncapsulateRect(rect);
+                {
+                    gui.state.EncapsulateRect(rect);
+                    gui.lastRect = rect;
+                }
```

Comment:
This breaks the `ErrorRow` of the ProjectPageView...
(without this change the size of the `ScrollArea` is set to the size of the (remaining) window (too small) and scrollbars are never shown...

@ShadowTheAge Any ideas how to fix either?

Reply:
ok, found a fix that seems to work for both `ErrorRow` and `ScrollArea`